### PR TITLE
Adopt new Swift API names of Voice iOS 6.0

### PR DESCRIPTION
**Pull request by bobiechen-twilio**
- [x] I acknowledge that all my contributions will be made under the project's license.

## Updates
- Adopt new Swifty API names
- Audio device updates

### CODE REVIEW

File: AudioDeviceExample/ViewController.swift [hunk 1]
```diff
@@ -256,8 +258,6 @@ extension ViewController: CXProviderDelegate {
 
     func provider(_ provider: CXProvider, perform action: CXStartCallAction) {
         NSLog("provider:performStartCallAction:")
-
-        audioDevice.isEnabled = false
```
Comment (piyushtank) [inline AudioDeviceExample/ViewController.swift:260]: Is this change needed?

Comment (bobiechen-twilio) [reply AudioDeviceExample/ViewController.swift:260]: I was following the same updates and audio device pattern from the test app

Comment (piyushtank) [reply AudioDeviceExample/ViewController.swift:260]: AudioDevice should be disabled by default. should we add an assert instead?

Comment (bobiechen-twilio) [reply AudioDeviceExample/ViewController.swift:260]: I would prefer showing the SDK users that they should now only use the CallKit provider callback to enable/disable audio device. We do have test for the default value in the source.

Comment (piyushtank) [reply AudioDeviceExample/ViewController.swift:260]: I didn't realize this is audio device example sample app.
